jgrapht-core/src/test/java/org/jgrapht/traverse/VertexTrackingTraversalListener.java

```diff
@@ -0,0 +1,53 @@
+package org.jgrapht.traverse;
```

Comment:
Can you add the standard file header here?  (See #823 for why this isn't happening automatically.)